gke/create_api_key.sh

```diff
@@ -34,4 +34,32 @@ KEY_STRING=$(gcloud alpha services api-keys get-key-string $API_KEY_NAME --forma
 touch /tmp/dc-website-api-key
 > /tmp/dc-website-api-key
 echo "$KEY_STRING" >> /tmp/dc-website-api-key
-gcloud secrets create maps-api-key --data-file=/tmp/dc-website-api-key
\ No newline at end of file
+gcloud secrets create maps-api-key --data-file=/tmp/dc-website-api-key
+
+# Create API key for Data Commons API
+gcloud alpha services api-keys create \
+  --display-name=mixer-api-key \
+  --allowed-referrers=$DOMAIN \
+  --api-target=service=api.datacommons.org
+
+API_KEY_NAME=$(gcloud alpha services api-keys list --filter='displayName=mixer-api-key' --format='value(name)')
+KEY_STRING=$(gcloud alpha services api-keys get-key-string $API_KEY_NAME --format='value(keyString)')
+
+touch /tmp/mixer-api-key
+> /tmp/mixer-api-key
+echo "$KEY_STRING" >> /tmp/mixer-api-key
+gcloud secrets create mixer-api-key --data-file=/tmp/mixer-api-key
+
+# Create API key for Generative Language API
+gcloud alpha services api-keys create \
+  --display-name=nl-palm-api-key \
+  --allowed-referrers=$DOMAIN \
+  --api-target=service=generativelanugage.googleapis.com
+
+API_KEY_NAME=$(gcloud alpha services api-keys list --filter='displayName=nl-palm-api-key' --format='value(name)')
+KEY_STRING=$(gcloud alpha services api-keys get-key-string $API_KEY_NAME --format='value(keyString)')
+
+touch /tmp/palm-api-key
+> /tmp/palm-api-key
+echo "$KEY_STRING" >> /tmp/palm-api-key
+gcloud secrets create palm-api-key --data-file=/tmp/palm-api-key
```

Comment:
nit: add newline at the end of both .sh file

Reply:
Done!